src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenJsonOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Text.Json;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen.DependencyInjection;
+
+/// <summary>
+/// Configures the <see cref="JsonSerializerOptions"/> to be used by <see cref="JsonSerializerDataContractResolver"/>.
```

Comment:
We have two implementations of `ISerializerDataContractResolver` (the other being `NewtonsoftDataContractResolver`), so rather than explicitly reference one in code, I would make this less specific.

Maybe something like:

> A class to specify options to use to configure JSON serialization for OpenAPI documents.